modules/test-utils/src/utils/layer.ts

```diff
@@ -0,0 +1,15 @@
+import {UniformValue} from '@luma.gl/core';
+import {Layer} from '@deck.gl/core';
+
+/**
+ * Extract uniform values set for a Layer in the underlying UniformBlock store
+ */
+export function getLayerUniforms(layer: Layer): Record<string, UniformValue> {
+  let uniforms = {};
+  const uniformBlocks = layer.getModels()[0]._uniformStore.uniformBlocks.values();
+  for (const block of uniformBlocks) {
+    uniforms = {...uniforms, ...block.uniforms};
+  }
```

Comment:
probably a trivial difference here, but slightly faster to assign than to spread:

```suggestion
  const uniforms = {};
  const uniformBlocks = layer.getModels()[0]._uniformStore.uniformBlocks.values();
  for (const block of uniformBlocks) {
    Object.assign(uniforms, block.uniforms);
  }
```

https://jsbench.me/t6ly66gt2a/1